pyflakes/test/test_other.py

```diff
@@ -1159,6 +1159,36 @@ def f(): global foo
         def g(): foo = 'anything'; foo.is_used()
         ''')
 
+    @skipIf(version_info < (3, 3), "Python >= 3.3 only")
```

Comment:
Only the last one isnt legal syntax on py27

`def foo(a, b, c=0, *args, d=0, **kwargs)`

All the others are fine on versions before 3.3, and should not be skipped.

However, the bigger problem is that these tests do not verify the changes to pyflakes.

These tests can run on master, and they all pass, as they are in fact very similar to other functions we have elsewhere in the test suite.

To test your changes, tests need to

1. invoke `Checker` with a `scope` set to a value other than `ModuleScope` .  This can be done with `self.flakes( ... , scope=FunctionScope)`

2. *and* need to pass `Checker` the equivalent of `body[0]` in the issue https://github.com/PyCQA/pyflakes/issues/338 , but that cant be done with `self.flakes(..)` currently.  You might change the harness so that `input` can be either an `ast.Node`, or a string.

3. Inspect the deadScopes , similar to `test_scope_class`, to ensure it has the expected structure , on any version of Python, and any differences between Python versions are recorded into the test suite so they do not flippantly change in the future.